### PR TITLE
Check chunk size right after loading chunk

### PR DESCRIPTION
**Pull request by dachengx**
**What is the problem / what does the code in this PR do**

Check the size of chunk when loading data.

**Can you briefly describe how it works?**

Add a decorator `check_chunk_n` for `strax.StorageBackend._read_chunk`.

**Can you give a minimal working example (or illustrate with a figure)?**

Please include the following if applicable:
  - Update the docstring(s)
  - Update the documentation
  - Tests to check the (new) code is working as desired.
  - Does it solve one of the open issues on github?

Please make sure that all automated tests have passed before asking for a review (you can save the PR as a draft otherwise).

### CODE REVIEW

**Comment by WenzDaniel** [comment]
Ah sorry, by size you mean number of chunks, sorry. Now I get it.

File: strax/chunk.py [hunk 1]
```diff
@@ -425,3 +426,18 @@ def _update_subruns_in_chunk(chunks):
             else:
                 subruns[subrun_id] = subrun_start_end
     return subruns
+
+
+@export
+def check_chunk_n(f):
+    @wraps(f)
+    def wrapper(self, *args, **kwargs):
+        # assume chunk_info is the second argument
```
Comment (WenzDaniel) [inline strax/chunk.py:435]: an we add a check that 'chunk_info' is in kwargs?

**Review by WenzDaniel — COMMENTED** [review]
Hej Dacheng looks good to me. Just one small comment, and a bigger one. Can you please add a test which fails and raises your warning? You can use our Records test plugin for that, but you will need to manually manipulate the chunk ones it is made. It will suffice to test it only for the file system backend

**Comment by dachengx** [comment]
Yes, @WenzDaniel . I added the test for `check_chunk_n`.

**Review by WenzDaniel — APPROVED** [review]
Thanks, looks good

**Pull request merged**